pom.xml

```diff
@@ -9,6 +9,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <bundle.symbolicname>com.google.cloud.tools.app.java</bundle.symbolicname>
```

Comment:
Maybe `.lib` instead of `.java`? Also, can you just use this directly in the plugin section instead of defining a property? I think it would be more clear as to what it's for.